cernopendata_client/search.py

```diff
@@ -136,3 +136,29 @@ def get_files_list(server=None, record_json=None, protocol=None, expand=None):
         ]
 
     return files_list
+
+
+def get_file_info_remote(recid):
+    """Return remote file information list for given record.
+
+    Returns a list of dictionaries containing (checksum, name, size,
+    uri) for each file in the record.  Note that 'name' property is
+    not stored remotely, but is calculated in this function for
+    convenience.
+    """
+    file_info_remote = []
+    record_json = get_record_as_json(server="http://opendata.cern.ch", recid=recid)
```

Comment:
Why `server` is hardcoded here?

Reply:
Need to propagate `server` variable; thought of doing that while refactoring.

Reply:
okay, makes sense 👍